ppr-ui/src/components/mhrRegistration/HomeOwners/AddEditHomeOwner.vue

```diff
@@ -682,6 +683,7 @@ export default defineComponent({
             TransWill.hasDeletedOwnersWithProbateGrantOrAffidavit() &&
             localState.owner.partyType === HomeOwnerPartyTypes.EXECUTOR) {
             localState.ownerGroupId = localState.owner.groupId
+            localState.owner.description = localState.owner.suffix
```

Comment:
So we are duplicating the suffix and putting into the description field?

Reply:
refactored to set when parsing owners on submit